app/javascript/src/apis/axios.js

```diff
@@ -41,9 +41,9 @@ const handleSuccessResponse = response => {
 const handleErrorResponse = (error, authDispatch) => {
   if (error.response?.status === 401) {
     authDispatch({ type: "LOGOUT" });
-    Toastr.error({ error: error.response?.data?.error });
+    Toastr.error(error.response?.data?.error);
   } else {
-    Toastr.error({ error: error.response?.data?.error || error.message });
+    Toastr.error(error.response?.data?.error || error.message);
```

Comment:
Made this change in order to respect the changes done in PR https://github.com/bigbinary/neeto-ui/pull/634 in neetoui.